xwiki-platform-core/xwiki-platform-repository/xwiki-platform-repository-server-ui/src/main/resources/ExtensionCode/ExtensionSheet.xml

```diff
@@ -131,7 +131,7 @@
           $doc.authors
         #end
       ##------- Active Installs --------
-      #if ($doc.getValue('id'))
+      #if ($doc.getValue('installedCount') &amp;&amp; $doc.getValue('showInstalledCount'))
```

Comment:
showInstalledCount might be null and when it's null if you behave as if it was true so you'd better do somehting like

    $doc.getValue('showInstalledCount') != 0